projects/plugins/debug-helper/CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.4.0] - 2022-07-06
+### Added
+- Added several enhancements to the Autoloader debugger module that make debugging easier. [#23927]
```

Comment:
May as well drop this one, since line 11 says how we added the whole Autoloader debugger module.